requirements.txt

```diff
@@ -28,3 +28,4 @@ hyperglot
 fontFeatures>=1.6.2
 vharfbuzz
 bumpfontversion
+nanoemoji
```

Comment:
best to require a relatively recent version (the latest is preferable), use `>=`